src/plugins_3rdparty/primer3/src/Primer3Task.h

```diff
@@ -196,6 +197,7 @@ class Primer3SWTask : public Task {
     Primer3Task* primer3Task = nullptr;
     int median;
     Primer3TaskSettings* settings;
+    bool ownsSettings;
```

Comment:
initialize fields with default values.

Reply:
What is the default value here?

Reply:
This default value will never be used and it only confuses the code.

Reply:
✓

Reply:
You may have multiple constructors far from the variable declaration and forget to initialize the field on some of them. If you initialize it by default the error will be clear: `null` is always better than some random value.

Most of other languages do it by default. With C++ & default memory allocator you need this boilerplate code. And having a good and safe code (like other languages do) is more important than side-effects of the boilerplate code it require.